WordPress/src/main/java/org/wordpress/android/ui/accounts/AbstractFragment.java

```diff
@@ -63,12 +63,12 @@ protected boolean onDoneEvent(int actionId, KeyEvent event) {
         return false;
     }
 
-    protected boolean didPressNextKey(int actionId, KeyEvent event) {
+    public static boolean didPressNextKey(int actionId, KeyEvent event) {
         return actionId == EditorInfo.IME_ACTION_NEXT || event != null && (event.getAction() == KeyEvent.ACTION_DOWN
                 && event.getKeyCode() == KeyEvent.KEYCODE_NAVIGATE_NEXT);
```

Comment:
Covered with https://github.com/wordpress-mobile/WordPress-Android/pull/5848#discussion_r116982837. Thanks and sorry!